src/Camera.ios.tsx

```diff
@@ -14,16 +13,18 @@ const Camera = React.forwardRef<CameraApi, CameraProps>((props, ref) => {
 
   React.useImperativeHandle(ref, () => ({
     capture: async () => {
-      return await CKCameraManager.capture({});
+      // @ts-ignore it was like this already
+      return await NativeCameraKitModule.capture({}, findNodeHandle(nativeRef.current ?? null));
```

Comment:
What was like this?

Reply:
Why not say it in the comment 😄?

Reply:
Should be fixed with 79dfe341899ebcf39e412973cd26538504d370d4